src/pages/Login/index.tsx

```diff
@@ -73,14 +96,107 @@ const Login = () => {
     };
 
     return (
-        <BoxBase>
-            <ButtonBase variant="contained" color="primary">
-                You are unauthorized
-            </ButtonBase>
-            <ButtonBase color="secondary" variant="contained" onClick={startLogin}>
-                Login With Google
-            </ButtonBase>
-        </BoxBase>
+        <SnackbarProvider>

```

Comment:
SnackbarProvider should be in the App.tsx

Reply:
![image](https://user-images.githubusercontent.com/58205909/121803480-0f4a6480-cc6c-11eb-8669-aec34dd311a3.png)
I think @thienduc12311  can use useSnackbar to implement easily.
![image](https://user-images.githubusercontent.com/58205909/121803512-402a9980-cc6c-11eb-95c6-84ac4ad04d40.png)